content/en/docs/profiles.md

```diff
@@ -0,0 +1,132 @@
+| [Order Lifetime](#order-lifetime) | 8 hours |
+| [Certificate Common Name](#certificate-common-name) | No |
+| [Key Encipherment KU](#key-encipherment-key-usage) | No |
+| [TLS Client Auth EKU](#tls-client-authentication-extended-key-usage) | No |
+| [Subject Key ID](#subject-key-identifier-extension) | No |
+| [Validity Period](#validity-period) | 160 hours |
+
+</div>
+
+# Selecting a Profile
+
+The process for selecting a profile is described in [this Internet-Draft](https://datatracker.ietf.org/doc/draft-aaron-acme-profiles/), which we plan to work with the IETF ACME Working Group to turn into a full RFC. Not all ACME Clients have implemented this draft, so the client you use may not yet be able to select a profile.
+
+In general, if you want to select a profile, you should:
+
+1. Read your ACME client's documentation to see if it support profile selection, and if it does, how to tell it which profile you want.
+2. Fetch the Let's Encrypt [production](https://acme-v02.api.letsencrypt.org/directory) or [staging](https://acme-staging-v02.api.letsencrypt.org/directory) directory object to see which profiles are available.
+3. Configure your desired profile within your ACME client.
+
+# Glossary
+
+## Validation Properties
+
+The following validation properties can be controlled by our profiles:
+
+### Pending Authorization Lifetime
+
+This is how long an ACME client has to complete a domain control validation challenge. The clock starts when the ACME Authorization object is created (generally as a result of a new Order being created), and is represented by the [`expires` timestamp](https://datatracker.ietf.org/doc/html/rfc8555#section-7.1.4) in the pending Authorization object. This value is restricted to [at most 30 days](https://github.com/cabforum/servercert/blob/main/docs/BR.md#322419-agreed-upon-change-to-website---acme) by the Baseline Requirements.
+
+### Authorization Reuse Period
+
+This is how long an already-validated Authorization can be reused by new Orders containing the same identifier. The clock starts when a challenge is successfully fulfilled, and is represented by the [`expires` timestamp](https://datatracker.ietf.org/doc/html/rfc8555#section-7.1.4) in the valid Authorization object. This value is restricted to [at most 398 days](https://github.com/cabforum/servercert/blob/main/docs/BR.md#421-performing-identification-and-authentication-functions) by the Baseline Requirements.
+
+### Order Lifetime
+
+This is how long an ACME client has to complete the whole process of ordering a new certificate: placing a new Order, fulfilling any pending Authorizations, and finalizing that Order. The clock starts when the new Order object is created, and is represented by the [`expires` timestamp](https://datatracker.ietf.org/doc/html/rfc8555#section-7.1.3) in the Order object.
+
+## Certificate Properties
+
+The following certificate properties can be controlled by our profiles:
```

Comment:
nit: Ending the line with a colon looks weird now that it's immediately followed by a second-level heading. Maybe something like this?

```suggestion
Below are descriptions of the certificate properties that can be controlled by our profiles.
```